packages/config/config/devices/0x0438/touch-thermostat-16a.json

```diff
@@ -0,0 +1,348 @@
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 16,
+			"unit": "A",
+			"defaultValue": 16
+		},
+		{
+			"#": "3",
+			"label": "Buzzer Vibration Level",
+			"description": "Set buzzer and vibration level.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 2,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Low",
+					"value": 1
+				},
+				{
+					"label": "High",
+					"value": 2
+				}
+			]
+		},
+		{
+			"#": "4",
+			"label": "Powerloss Recover Settings",
+			"description": "Recover settings after power loss / power reset.",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Disable",
+					"value": 0
+				},
+				{
+					"label": "Enable",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "6",
+			"label": "High Temperature Threshold",
+			"description": "Turns off relay and sends notification if temperature is above threshold. Allowable range: 0, 20-60",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 60,
+			"unit": "°C",
+			"defaultValue": 45
+		},
+
+		{
+			"#": "7",
+			"label": "Temperature Hysteresis",
+			"description": "Temperature control hysteresis. Allowable range: 5-20",
+			"valueSize": 1,
+			"minValue": 5,
+			"maxValue": 20,
+			"unit": "0.1 °C",
+			"defaultValue": 5
+		},
+
+		{
+			"#": "8",
+			"label": "Energy Save Mode",
+			"description": "0 - Off, user schedule used. 1 - On, use Energy_Save_Heating temperature set by THERMOSTAT_SETPOINT_SET",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Energy save mode off",
+					"value": 0
+				},
+				{
+					"label": "Energy save mode on",
+					"value": 1
+				}
+			]
+		},
+
+		{
+			"#": "9",
+			"label": "Floor Sensor Type",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 4,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "NTC/10K",
+					"value": 1
+				},
+				{
+					"label": "NTC/15K",
+					"value": 2
+				},
+				{
+					"label": "NTC/50K",
+					"value": 3
+				},
+				{
+					"label": "NTC/100K",
+					"value": 4
+				}
+			]
+		},
+
+		{
+			"#": "10",
+			"label": "Temperature Control Reference",
+			"valueSize": 1,
+			"minValue": 1,
+			"maxValue": 3,
+			"defaultValue": 1,
+			"allowManualEntry": false,
+			"options": [
+				{
+					"label": "Room sensor",
+					"value": 1
+				},
+				{
+					"label": "Floor sensor",
+					"value": 2
+				},
+				{
+					"label": "Room + Floor sensor",
+					"value": 3
+				}
+			]
+		},
+
+		{
+			"#": "11",
+			"label": "Room Temperature Correction",
+			"description": "Correct room temperature sensor. Allowable range: -10 - 10",
```

Comment:
Unnecessary description. No need to restate the range if it matches min/maxValue.